configuring/configuring-the-log-store.adoc

```diff
@@ -0,0 +1,83 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="configuring-lokistack-storage"]
+= Configuring the log store
+:context: configuring-the-log-store
+
+toc::[]
+
+You can configure a `LokiStack` custom resource (CR) to store application, audit, and infrastructure-related logs.
+
+include::snippets/loki-statement-snip.adoc[leveloffset=+1]
+
+
+// Loki sizing
+include::modules/loki-sizing.adoc[leveloffset=+1]
+
+
+// Loki object storage
+include::modules/logging-loki-storage.adoc[leveloffset=+1]
+
+// create object storage
+include::modules/logging-loki-storage-aws.adoc[leveloffset=+2]
+include::modules/logging-loki-storage-azure.adoc[leveloffset=+2]
+include::modules/logging-loki-storage-gcp.adoc[leveloffset=+2]
+include::modules/logging-loki-storage-minio.adoc[leveloffset=+2]
+include::modules/logging-loki-storage-odf.adoc[leveloffset=+2]
+include::modules/logging-loki-storage-swift.adoc[leveloffset=+2]
+
+//Loki stirage STS
+[id="installing-log-storage-loki-sts"]
+=== Deploying a Loki log store on a cluster that uses short-term credentials
+
```

Comment:
No level 3 headings (===) in any files

Reply:
this will be done as part of contentx because we have many instances of third level headings in the doc and changing the heading level here involves a lot of restructuring.